src/main/groovy/de/dkfz/config/ExecutionService.groovy

```diff
@@ -0,0 +1,15 @@
+package de.dkfz.config
+
+import de.dkfz.execution.jobs.Command
+
+/**
+ * Created by kaercher on 12.01.17.
+ */
+public interface ExecutionService {
+
+    public ExecutionService getInstance()
+
+    public void execute(Command command)
```

Comment:
Is this really the whole interface? I'd need to see how it works out in RoddyCore.
Consider renaming it to CommandExecutionService or so. It does not do more than that.
Also getInstance() in a non static manner makes NO sense! getInstance() is used to retrieve the singleton object of the class! Does this really work?